text.c

```diff
@@ -150,7 +150,6 @@ static Action *action_alloc(Text *txt);
 static void action_free(Action *a);
 /* logical line counting cache */
 static void lineno_cache_invalidate(LineCache *cache);
-static size_t lines_skip_forward(Text *txt, size_t pos, size_t lines, size_t *lines_skiped);
```

Comment:
`text_skip_forward` seems to be only used within text.c, hence there is no need to export it and the renaming is unnecessary.